lib/rules/cli/4.1.yaml

```diff
@@ -1298,6 +1298,7 @@
   :options:
     :all: --all=<value>
     :key_val: <value>
+    :l: -l <value>
```

Comment:
4.4.yaml also needs that (4.4 is no longer a soft link as there are incompatible change).
```
$ ls -l lib/rules/cli/4.*
-rw-rw-r--. 1 lxia lxia 38029 Mar 12 17:40 lib/rules/cli/4.1.yaml
lrwxrwxrwx. 1 lxia lxia     8 Dec  5 15:59 lib/rules/cli/4.2.yaml -> 4.1.yaml
lrwxrwxrwx. 1 lxia lxia     8 Dec  5 15:59 lib/rules/cli/4.3.yaml -> 4.2.yaml
-rw-rw-r--. 1 lxia lxia 38033 Mar 12 17:40 lib/rules/cli/4.4.yaml
lrwxrwxrwx. 1 lxia lxia     8 Mar 11 17:04 lib/rules/cli/4.5.yaml -> 4.4.yaml
```

Reply:
Right, i just noticed that 4.3 is all the way symlinked to 4.1 but not 4.4. Thanks, updated!